File: httpsnippet/src/test/resources/output/node/native/text-plain.js
var http = require("http");

var options = {"path":"/har","headers":{"content-type":"text/plain"},"hostname":"mockbin.com","method":"POST","port":80};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write(Hello World);
req.end();

File: httpsnippet/src/test/resources/output/python/python3/text-plain.py
import http.client

conn = http.client.HTTPConnection("mockbin.com", "80")

payload = "Hello World"

headers = {
	"content-type": "text/plain"
}

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))